core/integration/services_test.go

```diff
@@ -1175,10 +1175,13 @@ func TestFileServiceSecret(t *testing.T) {
 
 	httpSrv, httpURL := httpService(ctx, t, c, content)
 
-	//nolint:staticcheck // SA1019 We want to test this API while we support it.
-	secret := c.HTTP(httpURL, dagger.HTTPOpts{
+    // TODO: This is unsafe as it gets cached.
```

Comment:
In our various draft PRs for other changes, @vito actually just removed this test entirely because we don't need to support it anymore, I think we can do the same here

Reply:
Great! I forgot to ask in a comment, glad you beat me to it! 🙌